home-css/modifiction_nav_CL.css

```diff
@@ -94,4 +94,9 @@
   .comp-header .login-form p a{
     color:var(--orange);
     text-decoration: underline;
-  }
\ No newline at end of file
+  }
+
+  /* .comp-header .filter-form.active{
```

Comment:
remove commented code if it is not needed

Reply:
not needed i will remove it